implementations/rust/ockam/ockam_command/src/enroll/command.rs

```diff
@@ -50,21 +60,62 @@ impl EnrollCommand {
         initialize_identity_if_default(&opts, &self.identity);
         node_rpc(rpc, (opts, self));
     }
+
+    pub fn is_running() -> bool {
+        if let Ok(path) = CliState::default_dir() {
+            return Path::new(&path.join("enroll_touch_file")).exists();
+        }
+        false
+    }
 }
 
 async fn rpc(ctx: Context, (opts, cmd): (CommandGlobalOpts, EnrollCommand)) -> miette::Result<()> {
     run_impl(&ctx, opts, cmd).await
 }
 
+fn set_signal_handler(terminal: Terminal<TerminalStream<Term>>) {
+    let confirmation = Arc::new(AtomicBool::new(false));
+    ctrlc::set_handler(move || {
+        if confirmation.load(std::sync::atomic::Ordering::Relaxed) {
+            if CliState::delete().is_ok() {
+                let _ = terminal.write_line("Local Ockam configuration deleted");
+            }
```

Comment:
Let's not remove anything here, as this operation will delete all the state, including the stuff that was created before the user run the `enroll` command.

Reply:
#6061 suggests doing `ockam reset -y` when enroll is killed.
That is why I added this.